src/linux/cgroups2.hpp

```diff
@@ -76,6 +76,15 @@ Try<Nothing> move_process(const std::string& cgroup, pid_t pid);
 Try<std::string> cgroup(pid_t pid);
 
 
+// Get the processes inside of a cgroup.
+Try<std::set<pid_t>> processes(const std::string& cgroup);
+
+
+// Assign a process to a cgroup, by PID. This removes the process from its
+// current cgroup.
+Try<Nothing> assign(const std::string& cgroup, pid_t pid);
```

Comment:
there's the same function already with cgroups2::move_process..? assign or assign_process does seem like a better name, but did you mean to add two of the same function?